web/tools/webpack.config.ts

```diff
@@ -253,7 +253,6 @@ const createConfig = (
         },
         {
           test: /\.css$/,
-          include: /node_modules/,
```

Comment:
Why removed this?

Reply:
crashes css files. don't see any case why this is needed. everything works as expected without it